### DISCUSSION TITLE
Satpi DVB api only 1 channel

### DISCUSSION THREAD

**Discussion by jack8282** [Barracuda09/SATPI#140] [General]
lCan only open 1 channel second channel from the same transponder can not open. You have to give a stop and play on another channel then it starts. Or it cannot open 2 channels from the same transponder.  I made a log when asked for a channel that says this

Frontend: 0, Start filter PID: 5256  demux: 0  filter: 1 (data 0x81 mask 0xFF 0x00)
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 3
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 1 to 0, decrypting batch size 3
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 1
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 1 to 0, decrypting batch size 3
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 1
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 1 to 0, decrypting batch size 1
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 1
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 1 to 0, decrypting batch size 2
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 2
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 1 to 0, decrypting batch size 3
Nov  5 05:04:27 optimussos2plus local1.debug SatPI[602]: Frontend: 0, Parity changed from 0 to 1, decrypting batch size 2

**Comment by Barracuda09** [comment]
Hi @jack8282 

Did you build a new version from git? Or should I send you a new one?

**Reply by jack8282** [reply]
The version I have you made with dvb api oscam  1.5.472~g4b20e24 Enigma.  Send me the new version to test thanks a lot satpi_mipsel.tar.  Vu+ solo2.  They have my address iordan82@abv.bg

**Reply by jack8282** [reply]
I'm waiting for you to send me a new version of the mail with dvb api thank you very much.  libcryptо.so.1.1.0 I had mailed you :)

**Comment by Barracuda09** [comment]
Hi @jack8282 

Please checkout the current version, I have done some fixes for this problem.

**Reply by jack8282** [reply]
Vu+ solo2. They have my address iordan82@abv.bg   send bin

**Reply by Barracuda09** [reply]
I have send you a mipsel version

**Reply by jack8282** [reply]
Thank you so much
